prow-configurations/README.md

```diff
@@ -0,0 +1,40 @@
+# Prow Installation
+
+## Overview
+
+This folder contains the installation script and the set of configurations for Prow. 
+
+## Prerequisites
+
+- Kubernetes 1.10+ on GKE
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 
+- [gcloud](https://cloud.google.com/sdk/gcloud/)
+- openssl
+
+## Installation
+
+First, you need to set OAuth2 token that has read and write access to the bot account. You can set it as an environment variable named `OAUTH` or set it interactively during installation.
+
+Run the following script to start the installation process: 
+
+```bash
+./install-prow.sh
+```
+
+Installation script will accomplish the following steps to install Prow:
```

Comment:
`The installation script accomplishes the following steps to install Prow:` or `performs`